cloud/pkg/admissioncontroller/admission.go

```diff
@@ -317,8 +321,44 @@ func (ac *AdmissionController) registerWebhooks(opt *options.AdmissionOptions, c
 		},
 	}
 
+	// mutatingWebhook contains all the kubeedge related Mutating webhook
+	mutatingWebhook := admissionregistrationv1.MutatingWebhookConfiguration{
```

Comment:
The name is better focus on the ability of the webhook, like `offlineMigrationWebhook` above